scripts/scripts.js

```diff
@@ -183,9 +182,6 @@ async function loadEager(doc) {
   let pageType = 'CMS';
   if (document.body.querySelector('main .product-details')) {
     pageType = 'Product';
-    const sku = getSkuFromUrl();
-    window.getProductPromise = getProduct(sku);
-
     preloadFile('/scripts/__dropins__/storefront-pdp/containers/ProductDetails.js', 'script');
     preloadFile('/scripts/__dropins__/storefront-pdp/api.js', 'script');
     preloadFile('/scripts/__dropins__/storefront-pdp/render.js', 'script');
```

Comment:
Wouldn't it be too late to preload if it has already loaded?

Reply:
That said, we should preload the PDP initializer. I can add that.

Reply:
The intent of the comment was to suggest we preload files only when we know we will need them. And importing/using the initialize functions is the first indication that we are about to use these.

I suppose what I would like to see is in `loadEager` we have some way to simplify preloads like this so that we don't have to entrust this to the merchant/si. They should just be able to trust that when the page is product, or when product block is used, any necessary preloads are fired off.

Reply:
added preload for the PDP initializer of PDP